common/entryCreditBlock/commitchain.go

```diff
@@ -179,7 +184,7 @@ func (c *CommitChain) CommitMsg() []byte {
 	return p
 }
 
-// Return the timestamp
+// GetTimestamp returns the timestamp in milliseconds
```

Comment:
no, the timestamp is in nanoseconds, so it should be "returns a timestamp with millisecond precision"